website/cue/reference/releases/0.30.0.cue

```diff
@@ -0,0 +1,362 @@
+				the [upgrade
+				guide](/highlights/2023-05-23-0-30-0-upgrade-guide#socket-source-max-length) for
+				more details.
+				"""
+			breaking: true
+			pr_numbers: [17162]
+		},
+		{
+			type: "enhancement"
+			scopes: ["amqp sink"]
+			description: """
+				The `amqp` sink now allows configuration of the `content_encoding` and
+				`content_type` message properties via the new `properties` configuration option.
+				"""
+			contributors: ["arouene"]
+			pr_numbers: [17174]
+		},
+		{
+			type: "enhancement"
+			scopes: ["docker_logs source"]
+			description: """
+				The `docker_logs` source now supports usage of the `tcp://` scheme for the `host`
+				option. The connection is the same as-if the `http://` scheme was used.
+				"""
+			contributors: ["OrangeFlag"]
+			pr_numbers: [17124]
+		},
+		{
+			type: "enhancement"
+			scopes: ["releasing"]
+			description: """
+				Vector's distroless libc docker images (tags ending in `-distroless-libc`) are now
+				based on Debian 11 rather than Debian 10. This matches Vector's published Debian
+				images (tags ending in `-debian`).
+				"""
+			contributors: ["SIPR-octo"]
+			pr_numbers: [17160]
+		},
+		{
+			type: "enhancement"
+			scopes: ["aws_s3 source", "aws_s3 sink"]
+			description: """
+				The `aws_s3` source and `aws_s3` sink now have full support for codecs and can
+				receive/send any event type allowing `aws_s3` to be used as a transport layer
+				between Vector instances.
+				"""
+			pr_numbers: [17098]
+		},
+		{
+			type: "fix"
+			scopes: ["elasticsearch sink"]
+			description: """
+				The `elasticsearch` sink now uses the correct API to automatically determine the
+				version of the downstream Elasticsearch instance (when `api_version = "auto"`).
+				"""
+			contributors: ["syedriko"]
+			pr_numbers: [17227]
+		},
+		{
+			type: "enhancement"
+			scopes: ["tag_cardinality_limit transform", "observability"]
+			description: """
+				The `tag_cardinality_limit` now includes the `metric_name` field on logs it produces
+				to more easily identify the metric that was limited.
+				"""
+			contributors: ["nomonamo"]
+			pr_numbers: [17295]
+		},
+		{
+			type: "fix"
+			scopes: ["gcp_stackdriver_metrics sink"]
+			description: """
+				The `gcp_stackdriver_metrics` sink now correctly refreshes the authentication token before it expires.
+				"""
+			pr_numbers: [17297]
+		},
+		{
+			type: "enhancement"
+			scopes: ["http sink"]
+			description: """
+				HTTP-based sinks now log the underlying error if an unexpected error condition is
+				hit. This makes debugging easier.
+				"""
+			pr_numbers: [17327]
+		},
+		{
+			type: "fix"
+			scopes: ["observability"]
+			description: """
+				Vector's internal logs were updated to use "suppress" rather than "rate-limit" in the hopes that it makes it clearer that it is only Vector's log output that is being suppressed, rather than data processing being throttled.
```

Comment:
Strange how this is like the only long line

Reply:
Hmm, yeah, must have neglected to format that one.